### PR TITLE
Fabo/3532 validator missing image info

### LINKED ISSUE #3532: Add note about missing validator image to ValidatorPage

### PR DESCRIPTION
**Pull request by faboweb**
Closes #3532 

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
What about a more descriptive message like so?

**Comment by faboweb** [comment]
sounds good!

File: src/components/staking/PageValidator.vue [hunk 1]
```diff
@@ -34,29 +34,40 @@
...
+                </h5>
+              </div>
             </div>
           </div>
+          <span
+            v-if="!validator.picture || validator.picture === 'null'"
+            class="no-img-info"
+          >
+            Would you like us to display your logo?&nbsp;<a
+              href="mailto:contact@lunie.io"
```
Comment (jbibla) [inline src/components/staking/PageValidator.vue]: should use intercom instead of email

File: src/components/staking/PageValidator.vue [hunk 2]
```diff
@@ -34,29 +34,40 @@
...
+                  +{{ rewards.amount | fullDecimals | noBlanks }}
+                </h5>
+              </div>
             </div>
           </div>
+          <span
+            v-if="!validator.picture || validator.picture === 'null'"
+            class="no-img-info"
+          >
+            Would you like us to display your logo?&nbsp;<a
```
Comment (jbibla) [inline src/components/staking/PageValidator.vue]:
```suggestion
            Looks like we don't have this validator logo.<a
```

File: src/components/staking/PageValidator.vue (hunk 1 above)
Comment (jbibla) [inline src/components/staking/PageValidator.vue]: intercom instead?

File: src/components/staking/PageValidator.vue [hunk 3]
```diff
@@ -34,29 +34,40 @@
...
+              href="mailto:contact@lunie.io"
+              >Write us</a
```
Comment (jbibla) [inline src/components/staking/PageValidator.vue]:
```suggestion
              >Contact us</a
```

File: src/components/staking/PageValidator.vue (hunk 1 above)
Comment (Bitcoinera) [reply src/components/staking/PageValidator.vue]: I think you are right. Intercom is quicker and overall easier to use

**Comment by Bitcoinera** [comment]
Guys... I think it is better testing before merging

File: src/components/staking/PageValidator.vue (hunk 1 above)
Comment (jbibla) [reply src/components/staking/PageValidator.vue]: intercom works on desktop 😛 - there is likely a different problem

**Pull request merged**